src/main/java/seedu/address/model/schedule/Weight.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.model.schedule;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.address.model.util.WeightUnit.KILOGRAM;
+
+import seedu.address.model.util.WeightUnit;
+
+
+/**
+ * Represents a Schedule's Weight in the address book.
+ */
+public class Weight {
+    public static final String MESSAGE_INVALID_WEIGHT_STATUS = "Weight must be a positive number.";
+    /**
+     * Value of weight in kg.
+     */
+    private final double weight;
```

Comment:
I don't know which one is better, "value" (used in most classes), or "weight"

Reply:
To clarify, do you mean changing the variable name to value?

Reply:
yes. but actly doesnt really matter